.github/workflows/swift-toolchain.yml

```diff
@@ -1573,121 +1573,21 @@ jobs:
           path: ${{ github.workspace }}/BuildRoot/Library/libxml2-${{ inputs.libxml2_version }}/usr
 
   stdlib:
-    # TODO: Build this on macOS or make an equivalent Mac-only job
-    if: inputs.build_os == 'Windows'
-    needs: [compilers, cmark_gfm]
+    needs: [compilers]
```

Comment:
Compilers will need the CMark GFM to execute, why is the dependency removed?

Reply:
It should be though - we need to download and extract it. If we don't the compiler will not run.